### ISSUE TITLE
[Enhancment] issue with responsiveness of Apache AGE Viewer

### ISSUE THREAD

**Issue by Amr-Shams** [apache/age-viewer#134]
I have noticed that the responsiveness of Apache AGE Viewer is not being handled correctly. When I resize the browser window or access the website on a smaller screen device, the viewer's layout, and functionality become distorted and difficult to use.

Specifically, I have noticed the following issues:

The navigation menu and UI elements do not adjust to smaller screen sizes, resulting in overlapping and unreadable text.

Some buttons and links become challenging to click or access on smaller screens.
![image](https://user-images.githubusercontent.com/69462228/236354239-5a418a54-284e-4e6b-957d-cb28a0a46337.png)

**Comment by Mghrabi** [comment]
Assign me to this issue if you haven't already worked on it

**Comment by aybeedee** [comment]
Attempted a fix at the issue. Open to feedback, can make further improvements if needed.

**Comment by Nimra-1234** [comment]
Yeah, it has issues with the frontend in its updated version that is available on git further work is in process and soon will be resolved.
